plugins/score-card/README.md

```diff
@@ -31,8 +31,33 @@ You may drill down to the details of each score together with explanation why it
 
 ![Score Card Detail](./docs/.assets/score-card-detail.png)
 
+### EntityScoreCardTable
+
+Table that displays list of entities and their scores. This is different from ScoreCardTable as this component could be directly used in Entity Context within catalog as shown below:
```

Comment:
Just FYI: Now it's confusing IMHO. `ScoreCardTable` will be rename to `ScoreBoardTable`, see the comment below.

Reply:
I have added more description for `EntityScoreBoardTable`. `EntityScoreBoardTable` carries entity reference while fetching JSON, see [here](https://github.com/anand-tyagaraj/backstage-plugins/blob/80178a657e6962b3cb79fbc07f911324d5d8c678/plugins/score-card/src/components/EntityScoreBoardTable/EntityScoreBoardTable.tsx#L35C62-L35C68). That means `getAllScores` method has entityRef, using which it can read `all.json` dynamically from annotations. For example, see [here](https://github.com/anand-tyagaraj/score-card-samples/blob/7823a599773d07d7867b8899831860d2c42eaa75/catalogs/catalog-info-hello.json#L9) . Whereas  `ScoreCardTable` doesnt pass on entityRef, so it falls back to default implementation, i.e reading from `app-config.yaml`.